commit 1fb00098a87acb3a54424b6c2476c48b80379271 (refs/changes/26/2026/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-06-01T15:57:19-07:00 (2 years, 4 months ago)
    
    MANTA-3290 alarm metadata missing from MANTA-3195 commit
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 9d9f6e3..461a7de 100644
--- a/Makefile
+++ b/Makefile
@@ -118,7 +118,8 @@ release: all deps docs $(SMF_MANIFESTS)
 	@echo "Building $(RELEASE_TARBALL)"
 	@rm -rf $(PROTO)
 	@mkdir -p $(INSTDIR)
-	cp -r   $(TOP)/bin \
+	cp -r   $(TOP)/alarm_metadata \
+		$(TOP)/bin \
 		$(TOP)/build \
 		$(TOP)/cmd \
 		$(TOP)/config \
